turf/explode/_explode.py

```diff
@@ -0,0 +1,79 @@
+from typing import Dict, List, Sequence, TypeVar
+
+from turf.invariant import get_coords_from_features, get_coords_from_geometry
+
+from turf.helpers import feature, feature_collection, Point
+from turf.helpers import Feature, FeatureCollection, Geometry
+from turf.helpers import all_geometry_types
+
+from turf.utils.helpers import get_input_dimensions
+
+GeoJson = TypeVar("GeoJson", Feature, FeatureCollection, Geometry)
+
+
+def explode(features: GeoJson) -> FeatureCollection:
+    """
+    Takes a feature or set of features and returns all positions as {Point|points}.
+
+    # IMPORTANT! : in the original turf test cases, there were multiple
```

Comment:
I would remove this message from the main function and move it to the tests. 

This is a detail that the user doesn't have to know, and as far as he/she's concerned, this library works as expected, which is with points having 2 coordinates (x, y).